scylladb/sphinx-scylladb-theme
# Add custom scrollbar

Closes #259 

## How to test this PR

1. Clone this PR. For more information, see [Cloning pull requests locally](https://docs.github.com/en/github/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/checking-out-pull-requests-locally).

2. Enter the docs folder, and run:

```
make preview
````

3. Open http://127.0.0.1:5500/ with your favorite browser

4. Enter an internal page, and expand the "Contribute" dropdown:

The sidebar scrollbar should look like:

![image](https://user-images.githubusercontent.com/9107969/145183187-150d9a7d-1b90-4990-ba46-25a046edd027.png)
